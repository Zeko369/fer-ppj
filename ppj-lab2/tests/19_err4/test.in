KR_ZA 3 za
IDN 3 a
KR_OD 3 od
KR_DO 3 do
