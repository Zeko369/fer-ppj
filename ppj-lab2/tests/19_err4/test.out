err KR_DO 3 do
